lib/connection.js

```diff
@@ -384,6 +384,8 @@ class Connection extends EventEmitter {
 
     this.server.startSession(this, application, authStrategy, credentials,
       this._onSessionCreated.bind(this));
+
+    this.emit('handshakeRequest', applicationName, authStrategy, credentials);
```

Comment:
@belochub It won't be secure it user decides to log them, but they may have different trade-offs there.